### DISCUSSION TITLE
No workspace needed when the laptop lid is closed and a monitor is connected.

### DISCUSSION THREAD

**Discussion by rxmmah** [hyprwm/Hyprland#3390] [Q&A]
### how to make Hyprland workspaces start with number one on the dp-2?

**Comment by vdawg-git** [comment]
See https://wiki.hypr.land/Configuring/Binds/#switches for laptop switches.
Not sure what you mean with the second sentence.

Maybe you will also encounter this bug:
https://github.com/hyprwm/Hyprland/discussions/14114
